app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -334,6 +344,27 @@ def parse_persistent_volume(persistent_volume)
       new_result
     end
 
+    def parse_persistent_volume_claim(claim)
+      new_result = parse_base_item(claim)
+      new_result.merge!(
+        :desired_access_modes => claim.spec.accessModes.try(:join, ','),
+        :limits               => claim.spec.resources.limits.to_h.map { |k, v| "#{k}=#{v}" }.join(','),
+        :requests             => claim.spec.resources.requests.to_h.map { |k, v| "#{k}=#{v}" }.join(','),
```

Comment:
@zakiva take away these two because we want to implement them properly using the limit table.